modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -164,6 +162,27 @@ Kiali 1.73 uses the Jaeger Query API, which causes a longer response time depend
 
 . Verify the `istiod` pod logs and the `otel-collector` pod logs.
 
+[id="configuring-otel-collector-mtls-encrypted-namespace_{context}"]
+== Configuring the `OpenTelemetryCollector` in a mTLS encrypted Service Mesh member namespace
+
+All traffic is TLS encrypted when you enable Service Mesh `dataPlane` mTLS encryption.
+
+To enable the mesh to communicate with `OpenTelemetryCollector` service, disable the TLS `trafficPolicy` by applying a `DestinationRule` for the `OpenTelemetryCollector` service:
+
+.Example DestinationRule Tempo
+[source, yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: otel-disable-tls
+spec:
+  host: "otel-collector.bookinfo.svc.cluster.local"
+  trafficPolicy:
+    tls:
+  	mode: DISABLE
```

Comment:
Wrong formatting ( probably TAB instead of spaces )

Reply:
Copied from the GDoc. Don't know if that used space instead of TAB, or if it got converted from TAB to space.